stix2/sources/__init__.py

```diff
@@ -54,192 +59,191 @@ def make_id():
 FILTER_VALUE_TYPES = [bool, dict, float, int, list, str, tuple]
 
 
-class CompositeDataSource(object):
-    """Composite Data Source
-
-    Acts as a controller for all the defined/configured STIX Data Sources
-    e.g. a user can defined n Data Sources - creating Data Source (objects)
-    for each. There is only one instance of this for any python STIX 2.0
-    application
-
+class DataStore(object):
+    """
+    An implementer will create a concrete subclass from
+    this abstract class for the specific data store.
     """
+    def __init__(self, name="DataStore"):
+        self.name = name
+        self.id = make_id()
+        self.source = DataSource()
+        self.sink = DataSink()
 
```

Comment:
Yes, the `source` and `sink` properties should not be there. Also, the Abstract class should not be instantiated. `None` is a fine default.